app/src/main/java/org/geekhaven/ishare/activities/LoginActivity.kt

```diff
@@ -0,0 +1,100 @@
+package org.geekhaven.ishare.activities
+
+import android.content.Intent
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.support.v7.widget.Toolbar
+import android.text.TextUtils
+import android.view.View
+import android.widget.Button
+import android.widget.EditText
+import android.widget.ProgressBar
+import android.widget.Toast
+
+import com.google.android.gms.tasks.OnCompleteListener
+import com.google.android.gms.tasks.Task
+import com.google.firebase.auth.AuthResult
+import com.google.firebase.auth.FirebaseAuth
+
+import org.geekhaven.ishare.R
+
+class LoginActivity : AppCompatActivity() {
+
+    private var inputEmail: EditText? = null
+    private var inputPassword: EditText? = null
+    private var auth: FirebaseAuth? = null
```

Comment:
declare all these attributes as lateinit instead of initializing them as null. Also don't declare them as nullable.